### PR TITLE
[INLONG-4519][Manager] Add params and load node for Elasticsearch sink

### LINKED ISSUE #4519: [Feature]Manager support Elasticsearch sink
### Description

anager support Elasticsearch sink

### Use case

Support data writing to ES

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by LvJiancheng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-4519][Manager] Support Elasticsearch sink

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/incubator-inlong/issues) number)*

- Fixes #4519 

### Motivation

*Support Elasticsearch sink*

### Modifications

*Add  some utils and class about Elasticsearch.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/elasticsearch/ElasticsearchSinkOperation.java [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.function.Supplier;
+
+/**
+ * Elasticsearch sink operation, such as save or update Elasticsearch field, etc.
+ */
+@Service
+public class ElasticsearchSinkOperation implements StreamSinkOperation {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/elasticsearch/ElasticsearchSinkOperation.java:60]: Duplicate with `org.apache.inlong.manager.service.sink.es.ElasticsearchSinkOperation`, please reuse the old one.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/LoadNodeUtils.java [hunk 2]
```diff
@@ -328,6 +332,49 @@ public static SqlServerLoadNode createLoadNode(SqlServerSink sqlServerSink) {
         );
     }
 
+    /**
+     * Create SqlServer load node based on SqlServerSink
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/LoadNodeUtils.java:336]: The Java doc was incorrect.

**Review by healchow — CHANGES_REQUESTED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/elasticsearch/ElasticsearchSinkOperation.java (hunk 1 above)
Comment (LvJiancheng) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/elasticsearch/ElasticsearchSinkOperation.java]: done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/LoadNodeUtils.java (hunk 2 above)
Comment (LvJiancheng) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/LoadNodeUtils.java]: done

**Review by dockerzhang — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**